Resources/Prototypes/_Harmony/Catalog/Fills/Backpacks/duffelbag.yml

```
@@ -0,0 +1,11 @@
+- type: entity
+  parent: ClothingBackpackDuffelSyndicateBundle
+  id: ClothingBackpackDuffelSyndicateFilledBloodRedEnergyCarbineHarmony
+  name: blood-red energy carbine bundle
+  description: "A bundle containing the stolen and improved Blood-Red Energy Carbine. It comes with two spare Jailbroken Energy Cartridges"
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponBloodRedEnergyCarbineHarmony
+      - id: EnergyCarbineCartridgeSyndicateHarmony
+        amount: 2
```

You're one indentation level too high, the dashes to define a list in yaml isn't considered the list itself and shouldn't be indented.
```suggestion
    contents:
    - id: WeaponBloodRedEnergyCarbineHarmony
    - id: EnergyCarbineCartridgeSyndicateHarmony
      amount: 2
```